docs/schema/parameters-element-workflowconfig.md

```diff
@@ -43,29 +44,11 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Child elements
 
-<table>
-<colgroup>
-<col width="100%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td align="left"><p><span sdata="link"><a href="parameter-element-workflowconfig.md">Parameter Element (WorkflowConfig)</a></span></p></td>
-</tr>
-</tbody>
-</table>
+[Parameter Element (WorkflowConfig)](parameter-element-workflowconfig.md)
```

Comment:
Let's turn these into single column Markdown tables, like in this article:
docs/schema/parameters-element-in-method-bdcmetadataresource-schema.md